consensus/service/src/api/peer_service_error.rs

```diff
@@ -0,0 +1,26 @@
+use failure::Fail;
+use mc_transaction_core::tx::TxHash;
+
+/// Errors experienced when handling PeerAPI requests.
+#[derive(Debug, Fail)]
```

Comment:
I think we've established a pattern that new errors use `displaydoc` instead of `fail`, since that is no longer maintained.